### PR TITLE
Spelling typos fix

### PR DESCRIPTION
**Pull request by theodesp**
Fixes several typos reported by a Spell checker

### CODE REVIEW

File: orm/index_test.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ func TestCounterMultiKeyIndex(t *testing.T) {
 			expKeys:  [][]byte{encodeSequence(6), []byte("even")},
 			expError: false,
 		},
-		{ // id missmatch
+		{ // id miss-match
```
Comment (ruseinov) [inline orm/index_test.go]: it's actually mismatch

Comment (theodesp) [reply orm/index_test.go]: @ruseinov fixed Ta

File: orm/index_test.go [hunk 2]
```diff
@@ -164,7 +164,7 @@ func TestCounterMultiKeyIndex(t *testing.T) {
 			expKeys:  [][]byte{encodeSequence(6), []byte("even")},
 			expError: false,
 		},
-		{ // id miss-match
+		{ // id missmatch
```
Comment (ruseinov) [inline orm/index_test.go]: it's mismatch with one s. otherwise good

Comment (theodesp) [reply orm/index_test.go]: Silly me. Editing straight from Github!

**Comment by ruseinov** [comment]
Thanks!

**Pull request merged**